pagarme/former-kit
# Update dependencies

## Context
This PR aims to upgrade a lot of project dependencies, must of these dependencies are will suffer major updates which could break some interfaces.

## Checklist
- [x] Update major dependencies
- [x] Refactor dependencies usage


## Linked Issues
- [x] resolves #215 


## How to test this
1 - Download the PR branch
2 - Go to the project folder, delete `yarn.lock` and `node_modules`(just in case 😄 )
3 - Download the `former-kit-skin-pagarme` branch https://github.com/pagarme/former-kit-skin-pagarme/pull/156 and build it using:
```
yarn build
```
4 - Create a link in the `former-kit-skin-pagarme` using the following command
```
yarn link
```
5 - Link the `former-kit-skin-pagarme` in the `former-kit` project
```
yarn link former-kit-skin-pagarme
```
6 - Install all dependencies using the following command
```
yarn
```
7 - Start the _Storybook_ process using:
```
yarn storybook
```

The _Storybook_ should run without warnings.
